src/neo4j_genai/indexes.py

```diff
@@ -238,10 +238,11 @@ def drop_index_if_exists(
 
 def upsert_vector(
     driver: neo4j.Driver,
-    node_id: int,
+    id: int,
```

Comment:
Is there a specific reason to change the variable name?

Reply:
I'm wondering whether we want to have a separate function for uploading vectors to relationships instead. More of a separation of concerns thing

Reply:
Just because the function now works with nodes and vectors so keeping it as `node_id` might be a bit confusing. We can split it into 2 functions if you'd like they'd just be very similar and there'd be a bit of duplicated code.

Reply:
To me I think it makes sense to be separate as it's much more explicit and modular, also then we don't have breaking functionality for upsert_vector

Reply:
I think if we do that it would make sense to re-name `upsert_vector` though, otherwise people might get confused and assume it could be used to add embeddings to relationships.

Reply:
@willtai In order to avoid a breaking change I've left `upsert_vector` as it is other than some minor formatting changes and added an additional `upsert_vector_on_relationship` function. Let me know what you think. I was considering renaming `upsert_vector` to `upsert_vector_on_node` but this would also introduce a breaking change.

Reply:
I think that is good, as long we make it clear in the docs. Could you also please update the documentation for this? I think you can add the new `upsert_vector_on_relationship` there.

Could you also double check whether the example in the `upsert_vector` docstring is correct?

Reply:
Ah apologies I see that it's already added!

Reply:
I updated the docstring also, the example in there should be good now